libraries/chain/webassembly/eos-vm-oc/LLVMEmitIR.cpp

```diff
@@ -1258,12 +1281,11 @@ namespace LLVMJIT
 		}
 
 		if(module.tables.size()) {
-			current_prologue -= 8; //now pointing to LAST element
-			current_prologue -= 16*(module.tables.defs[0].type.size.min-1); //now pointing to FIRST element
 			auto tableElementType = llvm::StructType::get(context,{llvmI8PtrType, llvmI64Type});
-			defaultTablePointer = emitLiteralPointer((void*)current_prologue,tableElementType->getPointerTo(256));
+			llvm::Type* tableArrayTy = llvm::ArrayType::get(tableElementType, module.tables.defs[0].type.size.min);
+			defaultTablePointer = new llvm::GlobalVariable(*llvmModule, tableArrayTy, false, llvm::GlobalValue::ExternalLinkage, llvm::ConstantAggregateZero::get(tableArrayTy), getTableSymbolName());
```

Comment:
does this need to be marked ExternallyInitialized?

Reply:
I don't think so.  The external initialization should run after the the module is fully loaded.